src/components/Navigator/NavigatorCard.vue

```diff
@@ -461,9 +462,37 @@ export default {
       this.openNodes = openNodes;
       this.augmentRenderNodes({ uid: node.uid, exclude, include });
     },
+    toggleSiblings(node) {
+      const isOpen = this.openNodes[node.uid];
+      const openNodes = clone(this.openNodes);
+      const siblings = this.getSiblings(node.uid);
+      siblings.forEach(({ uid, childUIDs }) => {
+        if (!childUIDs.length) return;
+        if (isOpen) {
+          const children = this.getAllChildren(uid);
+          // remove all children
+          children.forEach((child) => {
+            delete openNodes[child.uid];
+          });
+          // remove the sibling as well
+          delete openNodes[uid];
+          // augment the nodesToRender
+          this.augmentRenderNodes({ uid, exclude: children.slice(1), include: [] });
```

Comment:
Why not excluding all children here instead?

Reply:
because getAllChildren returns the children + the current node. So I just slice off the current node.